src/mock.jl

```diff
@@ -18,6 +18,7 @@ macro mock(expr)
     # code below to have zero-overhead by only executing the original expression.
     result = quote
         if Mocking.activated()
+            @debug "Mocking activated, @mock macro expanding to `get_alternate` for target" $target
```

Comment:
Will fail for `baremodule`s. Doing `Mocking.@debug` would be safer. Do you find it useful to have this message for each call?